mula/scheduler/__init__.py

```diff
@@ -1,4 +1,37 @@
+import logging
+import os
+import threading
+
 from .app import App
 from .version import version
 
+logger = logging.getLogger(__name__)
 __version__ = version
+
+
+def shutdown(args: threading.ExceptHookArgs) -> None:
```

Comment:
It was more a stylistic choice since the placement of this piece of code would mean at the very top or the very bottom. Wanted to have the app.py solely have the App class in it. But I'm open to suggestions, my main concern would be legibility, easy to see what the app.py file contains and what it does.

Reply:
The only thought I head was that this looked like graceful shutdown logic that is needed when we do `app.run()`, so I thought why not set the `threading.excepthook` while initializing the `App` object? That would keep this setup away from any other code executing in the `scheduler` package (e.g. I can imagine that this hook is now globally set when we call alembic for migrations as well, although I do not expect this to give strange behavior!)

Reply:
Ah, I get what you're saying. Yes I agree that would be a more elegant implementation.